src/Arcus.BackgroundJobs.DataBricks/DatabricksJobMetricsAdditionalOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using GuardNet;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Arcus.BackgroundJobs.Databricks
+{
+    /// <summary>
+    /// Represents the additional options that the user can configure during the <see cref="IServiceCollectionExtensions.AddDatabricksJobMetrics"/> call.
+    /// </summary>
+    public class DatabricksJobMetricsAdditionalOptions
+    {
+        private int _intervalInMinutes = 5;
+
+        /// <summary>
+        /// Gets or sets the value which will be used when reporting the metric for finished Databricks job runs.
+        /// </summary>
+        public double MetricValue { get; set; } = 1;
```

Comment:
Hm this should always be 1, if this is related to my feedback [here](https://github.com/arcus-azure/arcus.backgroundjobs/pull/39#discussion_r455639324) then I was more referring to the metric name, not the value